graphql/image_test.go

```diff
@@ -65,6 +65,41 @@ func TestToolchains(t *testing.T) {
 	assert.Equal(t, testToolchain, utility.FromStringPtr(res[0].Name))
 }
 
+func TestEvents(t *testing.T) {
+	config := New("/graphql")
+	ctx := getContext(t)
+	testConfig := testutil.TestConfig()
+	testutil.ConfigureIntegrationTest(t, testConfig, "TestEvents")
+	require.NoError(t, testConfig.RuntimeEnvironments.Set(ctx))
+
+	// Returns the correct number of events according to the limit.
+	imageID := "ubuntu2204"
+	image1 := model.APIImage{
+		ID: &imageID,
+	}
+	res, err := config.Resolvers.Image().Events(ctx, &image1, 5, 0)
+	require.NoError(t, err)
+	assert.Len(t, res, 5)
+
+	// Does not return the same events in different pages.
+	allAMIAfter := map[string]struct{}{}
+	for _, event := range res {
+		allAMIAfter[utility.FromStringPtr(event.AMIAfter)] = struct{}{}
+	}
+	res2, err := config.Resolvers.Image().Events(ctx, &image1, 5, 1)
```

Comment:
instead of declaring a new variable, reuse `res`
```suggestion
	res, err = config.Resolvers.Image().Events(ctx, &image1, 5, 1)
```